testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/error/QRErroWindow.kt

```diff
@@ -0,0 +1,169 @@
+package com.rakuten.tech.mobile.testapp.ui.display.error
+
+import android.app.AlertDialog
+import android.content.Context
+import android.text.SpannableString
+import android.text.Spanned
+import android.text.TextPaint
+import android.text.method.LinkMovementMethod
+import android.text.style.ClickableSpan
+import android.view.LayoutInflater
+import android.view.View
+import androidx.databinding.DataBindingUtil
+import com.rakuten.tech.mobile.miniapp.testapp.R
+import com.rakuten.tech.mobile.miniapp.testapp.databinding.WindowQrCodeErrorBinding
+
+/**
+ * This QRErroWindow is the common class for qrcode/deeplink error.
+ */
+class QRErroWindow {
+
+    companion object {
+        private var mInstance: QRErroWindow? = null
```

Comment:
i think instance needs to be nullable to check it later, otherwise anyone can get a crash for wrong implementation
like if anyone implement this
```
QRErrorWindow().showMiniAppNoLongerExistError()
```
instead of

```
QRErrorWindow.getInstance(context).showMiniAppNoLongerExistError()
```